net/local/local_recvmsg.c

```diff
@@ -175,7 +175,7 @@ static void local_recvctl(FAR struct local_conn_s *conn,
   count = peer->lc_cfpcount;
   for (i = 0; i < count; i++)
     {
-      fds[i] = file_dup(peer->lc_cfps[i], 0,
+      fds[i] = file_dup(peer->lc_cfps[i], 0, /* TODO: WHat about this ? */
```

Comment:
ditto

Reply:
I left these TODO comments here so that someone can explain to me what these file_dup/file_dup2 calls are supposed to do. Is it OK to leave them like this or do I need to "fix" something.

Because now fd just points to a file, there is no need to make copies (like how file_dup2 works).

If it's fine to leave the file_dup() / file_dup2() calls as is, then I'll just remove the TODOs in the final patch.

Reply:
maybe, we should remove file_dupx